### PR TITLE
[useradmin] Sanitization process added to groups

### PR DESCRIPTION
**Pull request by skahler-pivotal**
The same sanitization process we do in users we run into in groups so the fix needs to be applied there additionally

### CODE REVIEW

File: apps/useradmin/src/useradmin/ldap_access.py [hunk 1]
```diff
@@ -326,7 +326,9 @@ def find_groups(self, groupname_pattern, search_attr=None, group_name_attr=None,
       group_filter = '(' + group_filter + ')'
 
     # Allow wild cards on non distinguished names
-    sanitized_name = ldap.filter.escape_filter_chars(groupname_pattern).replace(r'\2a', r'*')
+    sanitized_name = ldap.filter.escape_filter_chars(username_pattern).replace(r'\2a', r'*')
```
Comment (romainr) [inline apps/useradmin/src/useradmin/ldap_access.py]: Typo here, revert to groupname_pattern?

**Comment by romainr** [comment]
Nice!

And little nit, could you rename the commit like
[useradmin] Sanitization process added to groups
?

That's way the change log will look good ;)

File: apps/useradmin/src/useradmin/ldap_access.py (hunk 1 above)
Comment (skahler-pivotal) [inline apps/useradmin/src/useradmin/ldap_access.py]: yes you are correct that should be groupname_pattern

**Comment by skahler-pivotal** [comment]
commit renamed and reverted one of the changes

**Comment by romainr** [comment]
Awesome!

Last thing, do you think you could just merge the two commits into one?

e.g. git rebase -i HEAD~5
and put a 's' on the last one

that way we will just see one click commit!

**Pull request closed without merging**